CatFoodCalc/presets.py

```diff
@@ -0,0 +1,27 @@
+age = {
+    1: "kitten under 4 mo",
+    2: "kitten under 9 mo",
+    3: "kitten under 12 mo",
+    4: "adult cat",
+    5: "pregnant cat",
+    6: "lactating cat (under 3 kittens)",
+    7: "lactating cat (3+ kittens)",
+    8: "senior cat"
+}
+
+activity = {
+    "indoor": "neutered, inactive and indoor cats",
+    "outdoor": "active and outdoor cats"
+}
+
+kcal_per_kg = {
```

Comment:
Similar situation to https://github.com/tymik/catfoodcalc/pull/2/files#r451651255